src/kubernetes.cc

```diff
@@ -961,6 +961,18 @@ json::value KubernetesReader::FindTopLevelOwner(
   return FindTopLevelOwner(ns, GetOwner(ns, ref->As<json::Object>()));
 }
 
+const bool KubernetesReader::IsConfigured() const {
+  try {
+    json::value raw_node = QueryMaster(
+        std::string(kKubernetesEndpointPath) + "/nodes");
```

Comment:
It would be useful to minimize output by using fieldSelector (or limit=0 if that works)

https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#list-474